recipes/eabase/all/test_package/test_package.cpp

```diff
@@ -3,6 +3,22 @@
 #include <cstdlib>
 #include <iostream>
 
+#ifndef EA_COMPILER_IS_ANSIC
```

Comment:
What compiler did it not detect?

Is there a big ticket opened upstream for this?